### ISSUE TITLE
app.css not found

### ISSUE THREAD

**Issue by jhpedemonte** [maqetta/maqetta#2238]
When loading latest code see the following error in the console:

```
GET http://localhost:8081/maqetta/user/maqettaUser/ws/workspace/project1/maqetta/app.css 404 (Not Found)
```

@JonFerraiolo @peller @billreed63

**Comment by jhpedemonte** [comment]
It's odd that it has "maqetta" in the URL there.  I wonder if this has anything to do with the "maqetta" lib/package changes I made yesterday.

@childsb

**Comment by jhpedemonte** [comment]
OK, this is due to the values in **body_oam.json**:

```
        {
            "type": "css",
            "src": "app.css",
            "$library": "maqetta"
        },
```

Since _app.css_ is being loaded as part of the "maqetta" library, whose root was now changed (after ed1788a30bf13f4d6a2d06ffcdbf433de8d72450).

**Comment by JonFerraiolo** [comment]
-> @jhpedemonte so this doesn't fall through cracks

**Comment by jhpedemonte** [comment]
Fixed by b9e88648c11ee77e0bf2c00849542cd37ed611fa.
